src/DataStructures.jl

```diff
@@ -16,7 +16,9 @@ module DataStructures
                  eachindex, keytype, valtype
     import Base: iterate
 
-    using InteractiveUtils: methodswith
+    using OrderedCollections
+    using OrderedCollections: not_iterator_of_pairs, rehash!
```

Comment:
`rehash!` AFAICT is only used by *OrderedCollections*
so I don't think we need it to be used.
(or did I miss an usage?)

Reply:
https://github.com/JuliaCollections/DataStructures.jl/blob/bdd7aa034f569fbea7fef5d1998caf1863937528/src/dict_sorting.jl#L6 and https://github.com/JuliaCollections/DataStructures.jl/blob/bdd7aa034f569fbea7fef5d1998caf1863937528/src/dict_sorting.jl#L16

Reply:
Ah, 
I thought that file had been moved over.
I think it should be moved over.
It uses only Base and OrderedCollections types, no?

Reply:
Yes, you're absolutely right.

Oddly, these comments only show up if I access this page in certain ways. Weird GitHub bug?